frontend/src/components/G6Graph.svelte

```diff
@@ -115,6 +117,19 @@
 
         graph.on("nodeselectchange", (e) => dispatch("nodeselected", e));
 
+        // Enable keyboard manipulation
+        graph.on("keydown", (e: IG6GraphEvent) => {
+            if (e.key === "ArrowRight") {
+                graph.translate(-50, 0);
```

Comment:
50 is a constant that should be a variable somewhere instead of being hardcoded in this place.

Reply:
Thank you. Should be solved as of commit c12343f.